audit_exceptions/unstable_allowlist.json

```diff
@@ -5,6 +5,7 @@
   "ftgl": "2.1.3-rc",
   "libcaca": "0.99b",
   "librasterlite2": "1.1.0-beta",
+  "opentofu": "1.6.0-alpha",
```

Comment:
```suggestion
  "opentofu": "1.6.0-alpha2",
```

Reply:
I've reverted this. We need to leave the last number off since it produces a syntax error and would require us to bump it with every new alpha release.